lib/swoosh/adapters/amazon_ses.ex

```diff
@@ -90,7 +90,12 @@ defmodule Swoosh.Adapters.AmazonSES do
         {:ok, parse_response(body)}
 
       {:ok, code, _headers, body} when code > 399 ->
-        {:error, parse_error_response(body)}
+        case body do
+          "" ->
+            {:error, %{code: "ServiceUnavailable", message: "Empty response from SES"}}
```

Comment:
do you know what's the status code here? The `when code > 399` part indicates that it's at least 400 but what exactly?

Reply:
It wasn't in the logs. I'll include it in the logging perhaps?

Reply:
My best guess is that it times-out as there is no response, so 522? So maybe I should use timeout instead

Reply:
In any case, let's log it out to be sure. You can add a `Logger...` statement before this line. It might be 503 or smth else as well. So later maybe we can just check the status code and not even try to parse the body if we don't need to. Then we can also do an upstream PR.

Reply:
Added logging